mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule ExDoc.Mixfile do
   defp deps do
     [
       {:earmark, "~> 1.2"},
-      {:makeup_elixir, "~> 0.7"},
+      {:makeup_elixir, "~> 0.10.0"},
```

Comment:
Are you sure you want to depend on "0.10.0" and not on "0.10"?

Reply:
I can depend on "0.10", sure.